Makefile.toml

```
@@ -87,9 +117,10 @@ fi
 
 [tasks.fetch-sources]
 dependencies = ["setup"]
+script_runner = "bash"
 script = [
 '''
-for ws in sources packages variants/* tools/buildsys ; do
+for ws in sources packages variants/* tools/{buildsys,pubsys}; do
   [ -d "${ws}" ] || continue
   cargo fetch --locked --manifest-path ${ws}/Cargo.toml
 done
```

Since they share a workspace now:
```suggestion
for ws in sources packages variants/* tools ; do
  [ -d "${ws}" ] || continue
  cargo fetch --locked --manifest-path ${ws}/Cargo.toml
done
```